src/source/examples/get-ad-v5-example.rst

```diff
@@ -40,6 +40,7 @@
           "dateCreated": "2012-08-31T16:12:53Z",
           "dateLastUpdated": "2012-09-05T04:03:42Z",
           "vendorId": "OURSHOP-1423453-34",
+          "campaignId": 124152,
```

Comment:
same, I don't think this shows anywhere in our doc anymore (unless it's directly referenced in our freeform part, rather than the API reference)